lib/check.h

```diff
@@ -26,6 +26,7 @@
 #include "tokenize.h"
 #include "settings.h"
 #include "errorlogger.h"
+#include "cwe.h"
```

Comment:
Your call folks, I do not have enough visibility on the project to do a weighted choice. 

My only concern now is understand how to try that in a new branch without having to redo all changes.